components/FontSizeCounterInput.jsx

```diff
@@ -0,0 +1,37 @@
+import { useContext } from "react";
+import styles from "@/styles/components/font-size-counter-input.module.scss";
+import PageFeatures from "@/server/context";
+import CounterInput from "@/components/CounterInput";
+export default function FontSizeCounterInput() {
```

Comment:
pedantic warning! \n? UwU

Reply:
I know you start to enjoy finding those 🙌